apps/docs/app/utils/icons.server.ts

```diff
@@ -20,7 +20,7 @@ export const getIcon = async ({ category, fileName }: getIconArgs) => {
 };
 
 const getPathToSporIconsFolder = async (
-  currentPath: string = "."
+  currentPath: string = __dirname
```

Comment:
i dag lærte jeg forskjellen på disse to 😁 

`.` er working directoryen. Lokalt så funket det, siden det var der node_modules lå – men det var egentlig ikke det jeg tenkte det skulle være.
`__dirname` er det jeg egentlig burde brukt, som er hvilken mappe den filen som inneholder koden er i.